scripts/spark-container.dockerfile

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Build the test and build container for presto_cpp
+#
+FROM ghcr.io/facebookincubator/velox-dev:centos8 
+
+ARG SPARK_VERSION=3.5.1
+
+ADD scripts /velox/scripts/
+RUN wget https://archive.apache.org/dist/spark/spark-${SPARK_VERSION}/spark-${SPARK_VERSION}-bin-hadoop3.tgz
```

Comment:
The archive is pretty big, could you please download, untar and delete it in the same RUN command to remove the extra ~400MB from the image. 

~~The jar should be fine as that is used as is.~~ Actually I tested this and due to how layers work moving the jar/archive contents in another RUN step it effectively makes a copy. So please download, untar and move into final location in one step to minimize image size :)

Reply:
Updated as suggested. Thanks!